src/SprykerEco/Zed/PunchoutCatalogs/Communication/Table/PunchoutCatalogsConnectionsTable.php

```diff
@@ -0,0 +1,196 @@
+            true
+        );
+
+        if ($punchoutCatalogConnectionCollection->count() === 0) {
+            return [];
+        }
+
+        return $this->mapPunchoutCatalogConnections($punchoutCatalogConnectionCollection);
+    }
+
+    /**
+     * @module Company
+     *
+     * @param \Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnectionQuery
+     *
+     * @return \Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnectionQuery
+     */
+    protected function prepareQuery(PgwPunchoutCatalogConnectionQuery $connectionPropelQuery): PgwPunchoutCatalogConnectionQuery
+    {
+        return $connectionPropelQuery->joinWithCompanyBusinessUnit()
+            ->useCompanyBusinessUnitQuery()
+            ->joinCompany()
+                ->withColumn(SpyCompanyTableMap::COL_NAME, static::COL_COMPANY)
+            ->endUse();
+    }
+
+    /**
+     * @param \Propel\Runtime\Collection\ObjectCollection|\Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnection[] $punchoutCatalogConnectionCollection
+     *
+     * @return array
+     */
+    protected function mapPunchoutCatalogConnections(ObjectCollection $punchoutCatalogConnectionCollection): array
+    {
+        $punchoutCatalogConnectionRows = [];
+
+        foreach ($punchoutCatalogConnectionCollection as $punchoutCatalogConnection) {
+            $punchoutCatalogConnectionRows[] = $this->mapPunchoutCatalogConnection($punchoutCatalogConnection);
+        }
+
+        return $punchoutCatalogConnectionRows;
+    }
+
+    /**
+     * @param \Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnection $punchoutCatalogConnection
+     *
+     * @return array
+     */
+    protected function mapPunchoutCatalogConnection(PgwPunchoutCatalogConnection $punchoutCatalogConnection): array
+    {
+        $punchoutCatalogConnectionRow = $punchoutCatalogConnection->toArray();
+
+        $punchoutCatalogConnectionRow[static::ACTIONS] = $this->buildLinks($punchoutCatalogConnection);
+        $punchoutCatalogConnectionRow[static::COL_CREATED_AT] = $this->utilDateTimeService->formatDateTime($punchoutCatalogConnection->getCreatedAt());
+        $punchoutCatalogConnectionRow[static::COL_STATUS] = $this->getStatusLabel($punchoutCatalogConnection);
+        $punchoutCatalogConnectionRow[static::COL_FORMAT] = $this->generateLabel($punchoutCatalogConnection->getFormat(), 'label-secondary');
+        $punchoutCatalogConnectionRow[static::COL_COMPANY] = $punchoutCatalogConnection->getCompanyBusinessUnit()
+            ->getCompany()
+            ->getName();
+
+        return $punchoutCatalogConnectionRow;
+    }
+
+    /**
+     * @param \Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnection $punchoutCatalogConnection
+     *
+     * @return string
+     */
+    protected function buildLinks(PgwPunchoutCatalogConnection $punchoutCatalogConnection): string
+    {
+        $buttons = [];
+        $buttons[] = $this->generateEditButton('NOT_IMPLEMENTED', 'Activate');
```

Comment:
@asmarovydlo Why didn't you move 'Activate' to the glossary key, to have it translated?

Reply:
AFAIK we do not translate such stuff in Zed, e.g. here `Spryker\Zed\CmsGui\Communication\Table\CmsPageTable`

Reply:
> AFAIK we do not translate such stuff in Zed, e.g. here `Spryker\Zed\CmsGui\Communication\Table\CmsPageTable`

CmsPageTable is pretty old, actually. And I remember CC stream released translations for Zed UI. I'm not really sure, that while having translations functionality on Zed, we should continue adding not-translatable stuff.

Or table classes aren't covered by Zed UI translations?